### ISSUE TITLE
Support dry run for dataloaders to check if valid instances available

### ISSUE THREAD

**Issue by anwai98** [computational-cell-analytics/micro-sam#703]
I think it would make sense to perform a dry run on the dataloaders to check whether the instance labels are as expected (i.e. there has to be atleast one object instance to perform finetuning), else notify the user (even before allowing to finetune the model) about annotating some objects / adding sampler in the dataset, etc.

Ref: https://github.com/computational-cell-analytics/micro-sam/blob/master/micro_sam/training/training.py#L130

**Comment by anwai98** [comment]
This is taken care of at https://github.com/computational-cell-analytics/micro-sam/pull/705.
